ceph-lttng/README.md

```diff
@@ -0,0 +1,12 @@
+Builds an image with Ceph compiled with LTTnG. The image also has LTTnG tools
+for starting, stopping, and viewing the tracepoints.
+
+===================================================
+Quickstart
+===================================================
+
+Build Ceph with LTTnG and put it in an image:
+
+```bash
+CEPH_SRC=~/ceph ./build
+```
```

Comment:
Yes, it works (although I forgot to add a test to Travis CI). The flow is a little different from your other image builders but I basically add `lttng-tools` to the image by building another image on top of the image produced by `cephbuilder`.